docs/how-to/agent-service.rst

```diff
@@ -0,0 +1,72 @@
+Checkbox Agent Service
```

Comment:
I think I'd add systemd before service so it doesn't sound weird :D

Reply:
Fixed in [c0681c6](https://github.com/canonical/checkbox/pull/562/commits/c0681c6b3ef6bc1d79d3a05f4269e64a65800ec6)